build-image/openhabian.conf

```diff
@@ -89,4 +89,7 @@ storagecapacity=1024
 # Tailscale VPN
 #preauthkey=tskey-xxxxxxxxxxxxxxxxx
 
+# Custom log files
+#custom_log_files=("/var/log/foo.log" "/var/log/bar.log")
```

Comment:
I guess that format will confuse users. I don't mind but would prefer "file1 file2"

Reply:
Oh well. Users who don't understand that format probably don't know where or what the log files will be ahead of time anyway.